src/main/java/software/amazon/lambda/snapstart/CacheLambdaHandlerFields.java

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.amazon.lambda.snapstart;
+
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.Detector;
+import edu.umd.cs.findbugs.ba.ClassContext;
+import edu.umd.cs.findbugs.ba.XClass;
+
+import java.util.List;
```

Comment:
Don't we have some sort of linter or checkstyle? I don't see this `List` used anywhere on this file.